specs/development.json

```diff
@@ -465,8 +465,10 @@
                                         "type": "string",
                                         "title": "Campaign Providers",
                                         "description": "Where do you host your donations?",
-                                        "enum": [ "", "betterplace", "boost" ],
-                                        "default": ""
+                                        "enum": ["", "betterplace", "wecanhelp"],
```

Comment:
please leave boost in there, some communities and tools will have these values still set.

So we have a chance to migrate and remove boost later

Reply:
> please leave boost in there, some communities and tools will have these values still set.
> 
> So we have a chance to migrate and remove boost later

I have made the requested changes, please review and let me know if this fixes it!

Reply:
> > please leave boost in there, some communities and tools will have these values still set.
> > So we have a chance to migrate and remove boost later
> 
> I have made the requested changes, please review and let me know if this fixes it!
[https://github.com/freifunk/api.freifunk.net/pull/174](url)